models/users.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+//Users -- Model to work with an external API later on
+type Users struct {
+	Id     int    `csv,json:"id"`
```

Comment:
The acronyms should be in uppercase. Id -> ID

Reply:
Resolved :)